cassdegrees/ui/views/programs.py

```diff
@@ -6,6 +6,22 @@
 
 
 def create_program(request):
+    duplicate = request.GET.get('duplicate', False)
```

Comment:
You are either returning a string or a boolean value here - little inconsistent.

Reply:
Fixed it by setting it to 'false' string and then reassign duplicate variable as boolean True or False immediately after receiving it.